sys/tools/vnode_if.awk

```diff
@@ -319,7 +319,7 @@ while ((getline < srcfile) > 0) {
 
 	if (hfile) {
 		# Print out the vop_F_args structure.
-		printh("struct "name"_args {\n\tstruct vop_generic_args a_gen;");
+		printh("struct "name"_args {\n\tstruct vop_generic_args a_gen __subobject_member_used_for_c_inheritance;");
```

Comment:
I think this is correct and probably sufficient on its own without needing the other commit.